game/buildings.py

```diff
@@ -69,7 +57,11 @@ class TownHall(Building):
 
 class Granary(Building):
     MAX_LEVEL = 15
-    BASE_UPGRADE_TIME = timedelta(minutes=3)
```

Comment:
I think we should change `granary` to `warehouse` to standardize naming convention